tests/integration/reports/multi_table/_properties/test_multi_table_properties.py

```diff
@@ -48,3 +50,99 @@ def test_column_pair_trends_property():
 
     # Assert
     assert result == 0.49
+
+
+def test_coverage_property():
+    """Test the ``Coverage`` multi-table property end to end."""
+    # Setup
+    real_data, synthetic_data, metadata = load_demo(modality='multi_table')
+    coverage = Coverage()
+
+    # Run
+    result = coverage.get_score(real_data, synthetic_data, metadata)
+
+    # Assert
+    assert result == 0.96
+
+
+def test_coverage_property_with_progress_bar():
```

Comment:
can we either move these into their own files or put them in a class for each property that gets tested?

Reply:
Yes done ;)